package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-infinity",
-  "version": "0.2.0-beta.1",
+  "version": "0.2.0",
```

Comment:
We usually bump the version separate to a PR @davidgoli - can you please remove this file from the PR?